device/mellanox/x86_64-mlnx_msn4800-r0/ACS-MSN4800/sai_4800.xml

```diff
@@ -0,0 +1,19 @@
+<?xml version="1.0"?>
+<root>
+    <platform_info type="4800">
+
+        <!-- Device MAC address  -->
+        <device-mac-address>00:02:03:04:05:00</device-mac-address>
+
+        <!-- ISSU enabled -->
+        <issu-enabled>1</issu-enabled>
```

Comment:
please remove, currently not supported.

Reply:
do you mean remove:
`<issu-enabled>1</issu-enabled>`

or:
`<device-mac-address>00:02:03:04:05:00</device-mac-address>`

or both?

Reply:
I referred to the ISSU (warmboot is not currently supported for this platform)
As for the MAC, is it usually there ot this is a WA? if WA it cannot be in upstream code and might even cause a problem.

Reply:
i removed the <issu-enabled> tag
for the MAC - not a WA, it was always there